src/sentry/static/sentry/app/plugins/basePlugin.jsx

```diff
@@ -1,17 +0,0 @@
-import React from 'react';
-
-import Settings from 'app/plugins/components/settings';
-
-class BasePlugin {
-  constructor(data) {
-    Object.assign(this, data);
-  }
-
-  renderSettings(props) {
-    return <Settings plugin={this} {...props} />;
-  }
-}
-
-BasePlugin.DefaultSettings = Settings;
```

Comment:
Will removing this break anything?

Reply:
Looks like both uses of this were also updated.